tests/lib/mpsc_pbuf/testcase.yaml

```diff
@@ -5,3 +5,14 @@ tests:
       qemu_arc_em qemu_arc_hs qemu_cortex_a53 qemu_cortex_m0 qemu_cortex_m3
       qemu_cortex_r5 qemu_leon3 qemu_nios2 qemu_riscv32 qemu_riscv64 qemu_x86
       qemu_x86_64 qemu_xtensa
+    integration_platforms:
+      - native_posix
+
+  lib.mpsc_pbuf_concurrent:
+    tags: mpsc_pbuf
+    platform_allow: >
+      qemu_cortex_m3 qemu_x86
+    extra_configs:
+      - CONFIG_SYS_CLOCK_TICKS_PER_SEC=100000
```

Comment:
What's this for?  A 1MHz tick rate isn't really achievable on any of our platforms, hardware or no.